octree/basic_test.go

```diff
@@ -323,7 +502,14 @@ func TestBasicOctreePointcloudIngestion(t *testing.T) {
 
 	test.That(t, startPC.Size(), test.ShouldEqual, basicOct.Size())
 	test.That(t, startPC.MetaData(), test.ShouldResemble, basicOct.meta)
-	// TODO: Add iterate check of each point pointcloud to see if it is in octree (next JIRA ticket)
+
+	// Check all points from pointcloud exist in new basic octree
```

Comment:
[nit] "Check all points from pointcloud exist in new basic octree" is a bit confusing. Do you mean this?

Check if all points from the pointcloud startPC have been added to the octree basicOct